src/Classes/genre.rb

```diff
@@ -9,6 +9,6 @@ def initialize(_id, name)
 
   def add_item(item)
     @items << item unless @items.include?(item)
-    items.add_genre(self)
+    # items.add_genre(self)
```

Comment:
Please uncomment this line as it's required to add a genre for the item you're creating.